src/scss/footer.scss

```diff
@@ -0,0 +1,121 @@
+footer {
+	width: calc(100% - #{$navbar-width});
+	margin-left: $navbar-width;
+  color: $opfront-white;
+
+  input {
+      background-color: transparent;
+      color: inherit;
+      outline: none;
+      max-width: 200px;
+      border: $input-border-grey;
+      text-transform: none;
+  }
+}
+
+.subfooter {
+	width: 100%;
+  height: $subfooter-height;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: inherit;
+
+  &.newsletter{
+	  background-color: $opfront-dark-gray;
+  }
+
+  &.social {
+    background-color: $opfront-black;
+  }
+
+	.subfooter-content {
+		width: $content-width-desktop;
+    height: 100%;
+    color: inherit;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    .sitemap {
+      display: flex;
+
+      a {
+        color: inherit;
+        padding-right: 20px;
+        font-size: $sitemap-font-size;
```

Comment:
It would be nice if the sitemap items had an hover effect (social icons as well actually). Simply add: 
```
&:hover {
    color: $opfront-teal-something;
}
```
And don't forget to add a transition attribute to the ```a``` element:
```
transition: $default-transition-time;
```